client/app/domain/zone/record/domain-zone-record.controller.js

```diff
@@ -93,8 +93,9 @@ angular.module('App').controller(
         subDomain: punycode.toASCII(this.model.subDomainToDisplay || ''),
         target: this.model.target.value,
       })
-        .then((isOk) => {
-          this.recordConflicts = isOk;
+        .then((result) => {
```

Comment:
You can use destructuring:
```suggestion
        .then(({ recordCanBeAdded, conflictingRecords }) => {
```

In that way you are able to do::
```js
this.recordConflicts = recordCanBeAdded;
this.conflictingRecords = conflictingRecords;
```

Reply:
Thanks for the suggestion. This is done :)